gswikichat/vector_store_interface.py

```diff
@@ -1,24 +1,25 @@
 import os
 import json
 
-# from sentence_transformers import SentenceTransformer
 from tqdm import tqdm
 
 from haystack import Document  # , Pipeline
 from haystack.components.embedders import SentenceTransformersDocumentEmbedder
-# from haystack.components.embedders import SentenceTransformersTextEmbedder
 from haystack.document_stores.in_memory import InMemoryDocumentStore
-# from haystack.components.retrievers.in_memory import InMemoryBM25Retriever
 from haystack.components.retrievers.in_memory import InMemoryEmbeddingRetriever
-# from haystack.components.writers import DocumentWriter
 from haystack.document_stores.types.policy import DuplicatePolicy
+from haystack.components.preprocessors import DocumentSplitter
```

Comment:
Awesome! I was looking for those. Now I want to explore the whole preprocessor components list! This is so much better than my work.